src/librustc/lint/mod.rs

```diff
@@ -131,51 +131,104 @@ pub trait LintPass {
 // contains a few lint-specific methods with no equivalent in `Visitor`.
 pub trait LateLintPass: LintPass {
```

Comment:
Bleah I forgot about trait objects. Can the `Helper` trait have a single blanket impl proxying to the trait with the lifetime paramters?